unicef/giga-dagster
# Send Slack Release Notes

## What type of PR is this?

- `feat`: Commits that add a new feature

## Summary

What does this PR do

Enables sending Slack release notes when master is updated

## How to test
Upload a file via Giga Sync or the ADLS backend and verify if a Slack message is sent with the release details

## Link to Jira/Asana/Airtable task (if applicable)
https://app.clickup.com/t/86dtv9fdg
